exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+let message = "Save the planet earth!"
 console.log(message);
+let messageType = typeof message;
```

Comment:
Looks great. typeof message can be added directly into the console.log without creating additional variable. I like your work.